py3dtilers/Common/tiler.py

```diff
@@ -84,43 +83,17 @@ def get_output_dir(self):
         else:
             return self.args.output_dir
 
-    def write_geometries_as_obj(self, features, file_name):
-        obj_writer = ObjWriter()
-        obj_writer.add_geometries(features.get_features())
-        obj_writer.write_obj(file_name)
-
-    def change_projection(self, features, crs_in, crs_out):
-        transformer = Transformer.from_crs(crs_in, crs_out)
-        features.change_crs(transformer)
-
-    def create_tree(self, feature_list, create_lod1=False, create_loa=False, polygons_path=None, with_texture=False):
-        lod_tree = LodTree(feature_list, create_lod1, create_loa, polygons_path, with_texture)
+    def create_tree(self, feature_list, create_lod1=False, create_loa=False, polygons_path=None, with_texture=False, kd_tree_max=500):
+        lod_tree = LodTree(feature_list, create_lod1, create_loa, polygons_path, with_texture, kd_tree_max)
```

Comment:
why not rename LodTree() to create_tree ? It seems that create_tree() only invoke this method. 
Also, you can return LodTree() directly.

Reply:
`LodTree(...)` is the constructor of the `LodTree` class, thus it can't be renamed. However, this method could be deleted since it only creates the LodTree. The LodTree would be created in `create_tileset_from_geometries`. I won't do it in this PR to avoid confilcts with master branch, but I'll do this modification later.